.github/workflows/wheels.yml

```diff
@@ -212,16 +212,6 @@ jobs:
           pip install --upgrade setuptools pip wheel
           pip install cibuildwheel=="${{ env.cibuildwheel }}" cython=="${{ env.cython }}"
 
-      - name: install mac dependencies
```

Comment:
These were redundant - cibuildwheel ignores the host environment and uses its own pinned install, controlled via $CIBW_DEPENDENCY_VERSIONS